lib/oli_web/components/utils.ex

```diff
@@ -69,4 +70,23 @@ defmodule OliWeb.Components.Utils do
         false
     end
   end
+
+  attr :timezone, :string, required: true
+  attr :class, :string, default: nil
```

Comment:
I see that the `:class` attribute is defined with **default: nil**, but it’s never passed when the component is used. Would it make sense to remove it for now, since it doesn’t seem necessary and can always be added later if needed?